src/main/java/net/datafaker/providers/base/Currency.java

```diff
@@ -2,32 +2,29 @@
 
 /**
  * @since 0.8.0
+ * @deprecated since 2.2.0. For removal in 3.0.0 version. This faker is deprecated due to the existence
+ * of duplicate {@link Money} faker with similar methods. Use {@link Money} instead.
  */
+@Deprecated(since = "2.2.0", forRemoval = true)
 public class Currency extends AbstractProvider<BaseProviders> {
 
     public Currency(BaseProviders faker) {
         super(faker);
     }
 
+    /**
+     * @deprecated since 2.2.0. For removal in 3.0.0 version. Use {@link Money#currency} instead.
+     */
```

Comment:
If you're going to JavaDoc that much I'd also include: `@Deprecated(since = "2.2.0", forRemoval = true)`

Reply:
I think if the whole class is marked for removal, that  should be enough no?

Reply:
I guess, I'm not sure if inherit is the default. Figured it was best to be specific for users/implementers. Not a blocker I guess

Reply:
Make sense. Resolved.

thank you for the review